Core/Types/RelationshipDescriptor.cs

```diff
@@ -225,9 +231,12 @@ public override bool WithinBounds(CkanModule otherModule)
                 ?? false;
         }
 
-        public override List<CkanModule> LatestAvailableWithProvides(IRegistryQuerier registry, GameVersionCriteria crit, IEnumerable<CkanModule> toInstall = null)
+        public override List<CkanModule> LatestAvailableWithProvides(
+            IRegistryQuerier registry, GameVersionCriteria crit, IEnumerable<CkanModule> installed = null,
+            IEnumerable<CkanModule> toInstall = null
+        )
         {
-            return any_of?.SelectMany(r => r.LatestAvailableWithProvides(registry, crit, toInstall)).ToList();
+            return any_of?.SelectMany(r => r.LatestAvailableWithProvides(registry, crit, null, toInstall)).ToList();
```

Comment:
```suggestion
            return any_of?.SelectMany(r => r.LatestAvailableWithProvides(registry, crit, installed, toInstall)).ToList();
```

Looks like this should be `installed` instead of `null`?

Reply:
Good catch, a spot I missed in the refactoring.